File: tests/fixtures/projects/simple_ts/src/UserService.ts
export class UserService {
    constructor(private id: number) {}

    getUser(): User {
        return new User(this.id);
    }

    updateUser(user: User): void {
        console.log(`Updating user ${user.id}`);
    }
}

export class User {
    constructor(public id: number) {}
}

File: tests/fixtures/projects/simple_py/user_service.py
class UserService:
    def __init__(self, user_id: int):
        self.user_id = user_id

    def get_user(self) -> dict:
        return {"id": self.user_id}

    def update_user(self, user: dict) -> None:
        print(f"Updating user {user['id']}")
